pallets/schemas/README.md

```diff
@@ -0,0 +1,79 @@
+If you want to use a particular schema on a testnet, it is suggested that you use specific Schema Ids or publish the needed schema yourself.
+
+### Actions
+
+The Schemas pallet provides for:
+
+- Registering or proposing new Schemas.
+- Retrieving schemas by their Id or name.
+- Validating a Schema model.
+- Retrieving last registered Schema Id.
+
+## Interactions
+
+### Extrinsics
+
+| Name/Description                                                                                      | Caller                                          | Payment | Key Events                                                                                                                               | Runtime Added |
+| ----------------------------------------------------------------------------------------------------- | ----------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `set_max_schema_model_bytes`<br />Governance action to alter the maximum byte length of Schema models | Governance                                      | Tokens  | [`SchemaMaxSizeChanged`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaMaxSizeChanged) | 1             |
+| `propose_to_create_schema_v2`<br />Creates a Council proposal a new schema to the Frequency Council   | Token Account                                   | Tokens  | [`Proposed`](https://paritytech.github.io/polkadot-sdk/master/pallet_collective/pallet/enum.Event.html#variant.Proposed)                 | 66            |
+| `create_schema_via_governance_v2`<br />Governance called version of `create_schema_v3`                | Frequency Council                               | Tokens  | [`SchemaCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaCreated)               | 66            |
+| `create_schema_v3`<br />Creates a new Schema.                                                         | Mainnet: Governance<br />Testnet: Token Account | Tokens  | [`SchemaCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaCreated)               | 1             |
+| `propose_to_create_schema_name`<br />Creates a Council proposal to set the name of a Schema           | Token Account                                   | Tokens  | [`Proposed`](https://paritytech.github.io/polkadot-sdk/master/pallet_collective/pallet/enum.Event.html#variant.Proposed)                 | 1             |
+| `create_schema_name_via_governance`<br />Governance called to set the name of a Schema                | Frequency Council                               | Tokens  | [`SchemaNameCreated`](https://frequency-chain.github.io/frequency/pallet_schemas/pallet/enum.Event.html#variant.SchemaNameCreated)       | 66            |
```

Comment:
Did you mean to say "Governance call?"  If so I guess that's what you also meant above.